Dockerfile

```diff
@@ -0,0 +1,30 @@
+# syntax=docker/dockerfile:1
+
+FROM bitnami/minideb:latest AS build-bonfhir
```

Comment:
A mix of copilot recommendations & the base image for the `tomcat` container also being from `bitnami`.  🤷 
Felt like the _I'm feeling lucky_ button.

I'll switch to your proposition because it's 10mb lighter (compressed)